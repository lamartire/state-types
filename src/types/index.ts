export interface ITypes {
  [x: string]: string
}
