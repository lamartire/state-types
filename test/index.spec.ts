import types from '../src'

describe('types', () => {
  it('creates types hash with given prefix', () => {
    expect(types('@@PREFIX', ['foo', 'bar'])).toMatchSnapshot()
  })
})
