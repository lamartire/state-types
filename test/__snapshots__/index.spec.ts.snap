// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`types creates types hash with given prefix 1`] = `
Object {
  "bar": "@@PREFIX/bar",
  "foo": "@@PREFIX/foo",
}
`;
